sql/moz-fx-data-shared-prod/monitoring_derived/structured_distinct_docids_v1/query.py

```diff
@@ -14,15 +14,15 @@
     * EXCEPT (metadata),  -- Some tables have different field order in metadata
     metadata.header.x_source_tags,
     metadata.document_namespace,
-    metadata.document_type,
+    CONCAT(metadata.document_type, '_v', metadata.document_version) AS document_type,
```

Comment:
We need the versions here, because later for stable and live we'll join on `_TABLE_SUFFIX` which has the document_type and version.